configs/mysql/values.yaml

```diff
@@ -0,0 +1,15 @@
+threads: 1
+myqlDatabase: sbtest
+mysqlIterations: 1
+pushgatewayURL: http://pushgateway.monitoring:9091
+cloud: em
+instance: mysql
```

Comment:
Since this label conflicts with the pushgateway label `instance` this shows up as `exported_job`. But then the value comes up as `mysql-1` not sure why.

![Screenshot from 2021-05-03 13-43-11](https://user-images.githubusercontent.com/5815795/116854682-b53e9400-ac15-11eb-9d1f-5696cd8d1a45.png)

Reply:
Scratch above assumption. See https://github.com/kinvolk/benchmark-containers/pull/30#discussion_r624981706.